src/Symfony/Component/Config/Definition/Processor.php

```diff
@@ -15,6 +15,8 @@
  * This class is the entry point for config normalization/merging/finalization.
  *
  * @author Johannes M. Schmitt <schmittjoh@gmail.com>
+ *
+ * @final since version 4.1
```

Comment:
unrelated to this PR? If yes, should be removed.

Reply:
not sure :) https://github.com/symfony/symfony/pull/23888/files#diff-e321d9e71b12d99326a74385f020e2eeR66

see also https://github.com/symfony/symfony/pull/23888#discussion_r170668024

i think it's reasonable

Reply:
then: missing changelog/upgrade entries

Reply:
changelog entry added. Suggestion for an upgrade note?

Reply:
yes!
in *4.1: The ... class has been made final
in *5.0: The ... class has been made final

:)

Reply:
Done :)